.github/pull_request_template.md

```diff
@@ -1,7 +1,18 @@
  <!-- If this is your first PR, welcome! Please make sure you read the [contributing guidelines](../CONTRIBUTING.md). -->
 
- <!-- please add an icon to the title of this PR (see ../CONTRIBUTING.md) -->
- <!-- the icon will be either ⚠️ (:warning:, major or breaking changes), ✨ (:sparkles:, feature additions), 🐛 (:bug:, patch and bugfixes), 📖 (:book:, documentation or proposals), 💚 (`:green_heart:`, testing), 💎 (`:gem:`, refactor), 🔧 (`:wrench:`, dev tooling and chores) or 🌱 (:seedling:, minor or other) -->
+ <!-- Please label this pull request according to what type of issue you are addressing (see ../CONTRIBUTING.md) -->
+**What type of PR is this?**
+
+<!--
+Add one of the following kinds:
+/kind bug
+/kind documentation
+/kind feature
+/kind testing
+/kind refactor
+/kind dev-tooling
```

Comment:
https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/947#issuecomment-699152093

Reply:
looks like there is predefined list of labels in prow https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.yaml 

we can either use these existing set or create a PR to add new ones (probably not super easy)
```
/kind bug
/kind cleanup
/kind documentation
/kind feature
/kind design
/kind api-change
/kind deprecation
/kind failing-test
/kind flake
/kind regression
```

Reply:
If we add new ones those wouldn't be taken into account by release notes right? I think the only ones missing from here are kind dev-tooling and kind testing

Reply:
(and kind other)

Reply:
https://github.com/kubernetes/release/blob/master/pkg/notes/notes.go#L71

Reply:
my suggestion is to just add these ones that the release note tool can use

Reply:
I thought it was just reading the kind (as a text field)

Reply:
- testing can be either `feature`,  `failing-test` or `flake` 
 - dev-tooling can also be a `feature` or `cleanup`

Reply:
+1 on starting with the existing ones. Maybe we can try to get dev-tooling and testing added in parallel.

Reply:
made the changes to use existing ones